Projections/hydro_MacProjector.cpp

```diff
@@ -4,6 +4,7 @@
 
 #include <AMReX_MultiFabUtil.H>
 #include <AMReX_ParmParse.H>
+#include <AMReX_BC_TYPES.H>
```

Comment:
Since this is the only change in this file, is this necessary?

Reply:
oops, thanks for pointing that out, I've removed it.

(I had earlier put the outflow-correction routine in this file.)